src/map.html

```diff
@@ -6,126 +6,245 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link href="/css/animate.css" rel="stylesheet" />
   <link href="/css/semantic-ui.css" rel="stylesheet" />
+  <link rel="stylesheet" type="text/css" href="/css/nouislider.css">
```

Comment:
In styles.less, we can see:
```css
@import "nouislider.less";
```

This file will be imported into styles.less, and then compiled into our styles.css file.